Aspekt.Target/ClassUnderTest.cs

```diff
@@ -1,44 +1,44 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+//using System;
```

Comment:
Did you mean to leave all of this commented instead of deleting?

Reply:
Oh mostly I just use this project as an area to mess around and look at the IL, test with. It doesn't actually go into the project. I should find a better sandbox.